helm/templates/maelstro-backend-deployment.yaml

```
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "helm-maelstro.fullname" . }}-maelstro-backend-deployment
+  labels:
+    {{- include "helm-maelstro.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.maelstro_backend.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "helm-maelstro.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.maelstro_backend.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "helm-maelstro.selectorLabels" . | nindent 8 }}
+        org.geonetwork.service/name: {{ include "helm-maelstro.fullname" . }}-maelstro-deployment
+    spec:
+      {{- if .Values.datadir.git.url }}
+      initContainers:
+      {{- include "maelstro.bootstrap_maelstro_configuration" $ | nindent 6 }}
+      {{- end }}
+      {{- with .Values.maelstro_backend.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.maelstro_backend.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.maelstro_backend.securityContext | nindent 12 }}
+          image: "{{ .Values.maelstro_backend.image.repository }}:{{ .Values.maelstro_backend.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.maelstro_backend.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 8000
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: 8000
```

```suggestion
              port: http
```
Since you are referring a named port here: https://github.com/georchestra/maelstro/pull/29/files#diff-1945376e61ea7b2f2bc2ce876e069407f4d6e533c27252dc9ac650016c64ed12R39

If you change that, you need to change it in
- https://github.com/georchestra/maelstro/pull/29/files#diff-1945376e61ea7b2f2bc2ce876e069407f4d6e533c27252dc9ac650016c64ed12R49
- https://github.com/georchestra/maelstro/pull/29/files#diff-44fbc7570cd9e179b41164ef2f8632320fb88175523060c073f9ad56a64afae1R45
- https://github.com/georchestra/maelstro/pull/29/files#diff-44fbc7570cd9e179b41164ef2f8632320fb88175523060c073f9ad56a64afae1R49

---

putting back 'http'